packages/jest-jasmine2/src/__tests__/iterators.test.ts

```diff
@@ -6,8 +6,6 @@
  *
  */
 
-'use strict';
-
 describe('iterators', () => {
```

Comment:
These tests looks like they belong in `expect`, don't they?

/cc @pedrottimark

Reply:
Yes, I will make sure that `expect` covers these test cases.